Tutorials/whatiswot/7-JSON_Schema_in_Practice_2/7-JSON-Schema-Demo.txt

```diff
@@ -0,0 +1,25 @@
+In the previous video, we talked about JSON Schema Basics, in this video, we will continue with generic keywords, Schema composition, and Schema Conditionality. 
+
+Some of the annotation keywords used in JSON Schema are title, description, and default. None of these annotation words are required but enhance good practice for self-documenting Schemas.
+The title and description keywords must be strings.
+The default keyword specifies a default value. 
+
+The enum keyword is used to restrict a value to a fixed set of values. It must be an array with at least one element, where each element is unique. 
+
+The const keyword is used to restrict a value to a single value. 
+
+JSON Schema includes a few keywords for combining Schemas together, the most used ones are allOf, anyOf, oneOf, and not. 
+All of these keywords must be set to an array, where each item is a Schema. 
+
+To validate allOf, the given data must be valid against all of the given subSchemas. 
+
+To validate anyOf, the given data must be valid against any (one or more) of the given subSchemas. 
+
+To validate oneOf, the given data must be valid against exactly one of the given subSchemas. 
+
+The not keyword declares that an instance validates if it doesn’t validate against the given subSchema. 
```

Comment:
there seems to be 1 space missing in the indentation on the left? I might be wrong